ignite/handlers/param_scheduler.py

```diff
@@ -183,47 +252,338 @@ def simulate_values(cls, num_events: int, **scheduler_kwargs: Any) -> List[List[
             values.append([i, scheduler.optimizer_param_groups[0][scheduler.param_name]])
         return values
 
-    @classmethod
-    def plot_values(cls, num_events: int, **scheduler_kwargs: Mapping) -> Any:
-        """Method to plot simulated scheduled values during `num_events` events.
 
-        This class requires `matplotlib package <https://matplotlib.org/>`_ to be installed:
+class StateParameterScheduler(BaseParamScheduler):
```

Comment:
Maybe, we have to split this file into two as it is now ~1500 lines:
- ignite/handlers/param_scheduler.py
- ignite/handlers/state_param_scheduler.py
?

Also, it would be better to make class names shorter. For example typing `ExponentialStateParameterScheduler` is too much:
- StateParameterScheduler -> StatePScheduler ?
- LambdaStateParameterScheduler -> LambdaStatePScheduler ?
- ExponentialStateParameterScheduler -> ExpStatePScheduler ?
- MultiStepStateParameterScheduler -> MultiStepStatePScheduler ?
Other better names ?

Reply:
Yes it is now too long => will split the file.
Let's try `StatePS*`  to shorten class names